core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -990,7 +990,7 @@ public WorkflowModel terminateWorkflow(
             workflowStatusListener.onWorkflowTerminatedIfEnabled(workflow);
             Monitors.recordWorkflowTermination(
                     workflow.getWorkflowName(), workflow.getStatus(), workflow.getOwnerApp());
-
+            LOGGER.error("Workflow {} is terminated because of {}", workflowId, reason);
```

Comment:
Should this be logged at INFO level, since this is an expected result.

Reply:
yeah make sense.